rundeckapp/grails-spa/packages/ui-trellis/src/components/execution-log/logViewer.vue

```diff
@@ -478,9 +478,14 @@ export default class LogViewer extends Vue {
     }
 
     private handleNewLine(entries: Array<any>) {
+       const updatableProps: Array<keyof IEventViewerSettings> = ['nodeBadge', 'gutter', 'timestamps', 'lineWrap', 'command']
 
       for (const vue of entries) {
         // @ts-ignore
+          for (const prop of updatableProps) {
```

Comment:
What do you think about updating the log builder with the new options in `@Watch('settingsChange'` so it builds the entries with the current settings?

Reply:
Good catch, Changed this approach.